src/helpers/networkTypes.ts

```diff
@@ -4,6 +4,10 @@ export enum Network {
   mainnet = 'mainnet',
   optimism = 'optimism',
   polygon = 'polygon',
+  // Here to satisfy TS
+  ropsten = 'ropsten',
+  rinkeby = 'rinkeby',
+  kovan = 'kovan',
```

Comment:
this was removed here https://github.com/rainbow-me/rainbow/commit/97add29e885e328604607b8d7cd66f42aa203744 wondering if something happened while rebasing